app/src/components/Nav.js

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+
+import { Link } from "@reach/router";
+
+const Nav = () => {
+  return (
+    <div>
```

Comment:
Small thing, maybe use `<nav>` over `<div>` here. It's more semantic and supported on all modern browsers (via [canIuse](https://caniuse.com/?search=nav)).

Reply:
oh thank you so much!